### ISSUE TITLE
State persistence documentation provides an incorrect method

### ISSUE THREAD

**Issue by 0xpolarzero** [evmts/tevm-monorepo#968]
## Description

See [the example from the documentation at Clients > State persistence](https://tevm.sh/learn/clients/#state-persistence):

```javascript
import {createMemoryClient, createSyncPersister} from 'tevm'

// Client state will be hydrated and persisted from/to local storage
const clientWithLocalStoragePersistence = await createMemoryClient({
  persister: createSyncPersister({
    storage: localStorage
  })
})
```

`createSyncPersister` is not exported from `tevm`, although there is a `createSyncStoragePersister` exported from `tevm/sync-storage-persister`.

I guess either this should be linked in `tevm` or the example should be modified (with the correct name as well).

It would then be:
```javascript
import {createMemoryClient} from 'tevm'
import {createSyncStoragePersister} from 'tevm/sync-storage-persister'

// Client state will be hydrated and persisted from/to local storage
const clientWithLocalStoragePersistence = await createMemoryClient({
  persister: createSyncStoragePersister({
    storage: localStorage
  })
})
```

**Comment by roninjin10** [comment]
@0xpolarzero fixed this in #969. Did you know if you click this edit docs button you can directly edit the docs and put up a pr?
<img width="893" alt="image" src="https://github.com/evmts/tevm-monorepo/assets/35039927/561163dd-8de9-4c8f-ae94-70b3454fbe75">

**Comment by 0xpolarzero** [comment]
@roninjin10 Yes, I was initially going to edit directly with a PR, but I was not sure if the documentation was right, and it should be exported from `tevm`, or the opposite.
